package.json

```diff
@@ -36,6 +36,7 @@
     "dist:svg:rapid": "svg-sprite --symbol --symbol-dest . --shape-id-generator \"rapid-%s\" --symbol-sprite dist/img/rapid-sprite.svg \"svg/rapid-sprite/**/*.svg\"",
     "dist:svg:roentgen": "svg-sprite --symbol --symbol-dest . --shape-id-generator \"roentgen-%s\" --symbol-sprite dist/img/roentgen-sprite.svg svg/roentgen/*.svg",
     "dist:svg:temaki": "svg-sprite --symbol --symbol-dest . --shape-id-generator \"temaki-%s\" --symbol-sprite dist/img/temaki-sprite.svg node_modules/@rapideditor/temaki/icons/*.svg",
+    "dist:svg:overture": "shx cp svg/overture/* dist/img/",
```

Comment:
I think it would be ok to just put the svg into dist/img rather than having this command to copy it over.

Reply:
Yup, that makes sense! No need to have it be off by itself. Will fix.